module/data/item/templates/physical-item.mjs

```diff
@@ -109,6 +109,10 @@ export default class PhysicalItemTemplate extends SystemDataModel.mixin(
                       integer: true,
                       label: "ED.Item.General.arbitraryStep"
                   } ),
+                  action: new fields.StringField( {
```

Comment:
This should use `ED4E.action` as `choices`

Reply:
changed it together with the hbs.